tests/Scripts/stray_baselines.py

```diff
@@ -10,14 +10,12 @@
 
 if __name__ == "__main__":
     try:
-        available_tests = set(
-            map(
-                lambda x: x.decode("utf-8"),
-                subprocess.check_output(
-                    ["btest", "-l", "-c", TEST_DIR + "/btest.cfg"]
-                ).splitlines(),
-            )
-        )
+        available_tests = {
+            x.decode("utf-8")
```

Comment:
What exactly is happening with this `x.decode` here? `x` isn't set to anything, so it's not just decoding a string.

Reply:
The part in `{..}` is a set comprehension, `x` is bound via `for x in ...` on the line below.